test/oscar/compiler/doubleDivision.oscar

```diff
@@ -0,0 +1,3 @@
+def main() => unit = {
+  println(1.1 / 2.);
```

Comment:
There's an edge case sometimes with doubles. You have to compare in a delta like this:

(1.1 / 2. > 0.549) && (1.1 / 2. < 0.551)